src/Symfony/Component/Routing/Loader/YamlFileLoader.php

```diff
@@ -243,6 +249,12 @@ protected function validate($config, string $name, string $path)
         if ($extraKeys = array_diff(array_keys($config), self::$availableKeys)) {
             throw new \InvalidArgumentException(sprintf('The routing file "%s" contains unsupported keys for "%s": "%s". Expected one of: "%s".', $path, $name, implode('", "', $extraKeys), implode('", "', self::$availableKeys)));
         }
+        if (isset($config['alias'])) {
+            if (1 === \count($config)) {
+                return;
```

Comment:
```suggestion
                // No need to continue, if an alias is set, it's the only desired key
                return;
```

Add a comment explaining why there is a return here :)